EtoFormsExample/Program.fs

```diff
@@ -1,19 +1,18 @@
 ﻿open Eto
 open Eto.Forms
 open Eto.Drawing
-open EtoUtils
 
 let app = new Application()
 
-let form = new Form(Title="Curves", TopMost=true, Size=Size(640, 480))
+let form = new Form(Title="Curves", Size=Size(640, 480))
```

Comment:
have to remove topmost because that property doesn't exist in this version of eto.forms